crates/ruff_python_parser/src/token.rs

```diff
@@ -89,6 +89,9 @@ impl Token {
                 | TokenKind::FStringStart
                 | TokenKind::FStringMiddle
                 | TokenKind::FStringEnd
+                | TokenKind::TStringStart
```

Comment:
We should update the documentation here to clarify that this includes t-strings and review all usages because t-string tokens feel like a string, they're not at runtime.